models/marts/core/dim_customers.sql

```diff
@@ -0,0 +1,27 @@
+with customers as (
+    select * from {{ ref('stg_customers')}}
+),
+orders as (
+    select * from {{ ref('stg_orders')}}
```

Comment:
with a little refactoring of `fct_orders` and then ref'ing that here, you can get all the data points you need (including customer total payment amount)

Reply:
Updated to refactor fct_orders and then ref'ing that in dim_customers